source/agora/test/Base.d

```diff
@@ -324,6 +324,7 @@ public NetworkT makeTestNetwork (NetworkT : TestNetworkManager)
                 retry_delay : 100, // msecs
                 max_retries : 20,
                 min_listeners : nodes - 1,
+                data_dir : format("%s/%s%d", ".cache", "ledger.dat", idx)
```

Comment:
Unittests should never do IO, which this does.

Reply:
just config not FilIO But deleted